modules/basis/basis-datasource/samples/kube.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dynamic-db
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dynamic-db
+  template:
+    metadata:
+      labels:
+        app: dynamic-db
+    spec:
+      containers:
+        - name: dynamic-db
+          image: radiatorspring/dynamic-db:latest
+          env:
+            - name: DefaultDB_url
```

Comment:
How about `DEFAULT_DB_URL` instead of `DefaultDB_url`?

Reply:
The way the code currently works is it takes the name of the JNDI resource and checks for env variables with names starting with the JNDI name, so I think we shouldn't change it, because by default everyone uses jdbc/DefaultDB and url is with small case because we use it as is in our code